test/test_checks.py

```diff
@@ -1,6 +1,21 @@
+import subprocess
+
+import flake8
```

Comment:
This repo seems to sort imports alphabetically not by first/third/local modules (e.g. `os` comes after `flake8_quotes`). Please move these new imports into the existing grouping